src/Microsoft.AspNet.Cors.Core/CorsResult.cs

```diff
@@ -0,0 +1,179 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.AspNet.Cors.Core
+{
+    /// <summary>
+    /// Results returned by <see cref="ICorsEngine"/>.
+    /// </summary>
+    public class CorsResult
+    {
+        private long? _preflightMaxAge;
+
+        /// <summary>
+        /// Gets a value indicating whether the result is valid.
+        /// </summary>
+        public bool IsValid
```

Comment:
i'm confused about https://github.com/aspnet/CORS/pull/3 and https://github.com/aspnet/CORS/pull/4 - is one of them out of date? do we still have ErrorMessages on the CorsResult?